src/test/java/uk/gov/dhsc/htbhf/utils/WiremockResponseTestDataFactory.java

```diff
@@ -29,84 +41,18 @@ public static String aValidVoucherEntitlement() {
-                "      \"ENTRY_DATE\" : \"19/05/2008\",\n" +
-                "      \"BLPU_STATE_DATE\" : \"19/05/2008\",\n" +
-                "      \"LANGUAGE\" : \"EN\",\n" +
-                "      \"MATCH\" : 1.0,\n" +
-                "      \"MATCH_DESCRIPTION\" : \"EXACT\"\n" +
-                "    }\n" +
-                "  }, {\n" +
-                "    \"DPA\" : {\n" +
-                "      \"UPRN\" : \"321324\",\n" +
-                "      \"UDPRN\" : \"51094466\",\n" +
-                "      \"ADDRESS\" : \"114 RATCLIFFE COURT, CHIMNEY STEPS, BRISTOL, " + postcode + "\",\n" +
-                "      \"BUILDING_NAME\" : \"114 RATCLIFFE COURT\",\n" +
-                "      \"THOROUGHFARE_NAME\" : \"CHIMNEY STEPS\",\n" +
-                "      \"POST_TOWN\" : \"BRISTOL\",\n" +
-                "      \"POSTCODE\" : \"" + postcode + "\",\n" +
-                "      \"RPC\" : \"2\",\n" +
-                "      \"X_COORDINATE\" : 359913.0,\n" +
-                "      \"Y_COORDINATE\" : 172758.0,\n" +
-                "      \"STATUS\" : \"APPROVED\",\n" +
-                "      \"LOGICAL_STATUS_CODE\" : \"1\",\n" +
-                "      \"CLASSIFICATION_CODE\" : \"RD06\",\n" +
-                "      \"CLASSIFICATION_CODE_DESCRIPTION\" : \"Self Contained Flat (Includes Maisonette / Apartment)\",\n" +
-                "      \"LOCAL_CUSTODIAN_CODE\" : 116,\n" +
-                "      \"LOCAL_CUSTODIAN_CODE_DESCRIPTION\" : \"BRISTOL CITY\",\n" +
-                "      \"POSTAL_ADDRESS_CODE\" : \"D\",\n" +
-                "      \"POSTAL_ADDRESS_CODE_DESCRIPTION\" : \"A record which is linked to PAF\",\n" +
-                "      \"BLPU_STATE_CODE\" : \"2\",\n" +
-                "      \"BLPU_STATE_CODE_DESCRIPTION\" : \"In use\",\n" +
-                "      \"TOPOGRAPHY_LAYER_TOID\" : \"osgb1000002529045129\",\n" +
-                "      \"PARENT_UPRN\" : \"321210\",\n" +
-                "      \"LAST_UPDATE_DATE\" : \"10/02/2016\",\n" +
-                "      \"ENTRY_DATE\" : \"19/05/2008\",\n" +
-                "      \"BLPU_STATE_DATE\" : \"19/05/2008\",\n" +
-                "      \"LANGUAGE\" : \"EN\",\n" +
-                "      \"MATCH\" : 1.0,\n" +
-                "      \"MATCH_DESCRIPTION\" : \"EXACT\"\n" +
-                "    }\n" +
-                "  }\n" +
-                "  ]\n" +
-                "}\n" +
-                "  ";
+        String template = getResponseTemplate("postcode-lookup-2-results.json");
+        return template.replace(DEFAULT_POSTCODE, postcode);
+    }
+
+    private static String getResponseTemplate(String templateName) {
+        return RESPONSE_TEMPLATES.computeIfAbsent(templateName, key -> {
+            try (InputStream stream = WiremockResponseTestDataFactory.class.getClassLoader().getResourceAsStream("wiremock/mappings/" + templateName)) {
+                return IOUtils.toString(stream, StandardCharsets.UTF_8);
```

Comment:
You can do this directly from the JDK now if you didn't want another import, you can use `String content = Files.readString(Paths.get(path));`.